src/themes/teams/components/Dropdown/dropdownMessageNoResultsStyles.ts

```diff
@@ -0,0 +1,13 @@
+import ListItem from '../../../../components/List/ListItem'
+import { DropdownMessageNoResultsProps } from '../../../../components/Dropdown/DropdownMessageNoResults'
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+
+const dropdownSelectedItemStyles: ComponentSlotStylesInput<DropdownMessageNoResultsProps, any> = {
+  root: ({ variables: v }): ICSSInJSStyle => ({
+    [`&.${ListItem.className}`]: { backgroundColor: v.listItemBackgroundColor },
+
+    fontWeight: 'bold',
+  }),
+}
```

Comment:
We have a task for cleaning up styles in `Dropdown`, we should separate variables between components and them meaningful.